rpm.mk

```diff
@@ -0,0 +1,171 @@
+	@command -v fpm > /dev/null || sudo gem install fpm --no-ri --no-rdoc
+	@ssh -o StrictHostKeyChecking=no git@github.com || true
+
+rpm-herokuish:
+	rm -rf /tmp/tmp /tmp/build $(HEROKUISH_RPM_PACKAGE_NAME)
+	mkdir -p /tmp/tmp /tmp/build
+
+	@echo "-> Cloning repository"
+	git clone -q "https://github.com/$(HEROKUISH_REPO_NAME).git" \
+		--branch "v$(HEROKUISH_VERSION)" /tmp/tmp/herokuish > /dev/null
+	rm -rf /tmp/tmp/herokuish/.git /tmp/tmp/herokuish/.gitignore
+
+	@echo "-> Copying files into place"
+	mkdir -p "/tmp/build/var/lib"
+	cp -rf /tmp/tmp/herokuish /tmp/build/var/lib/herokuish
+
+	@echo "-> Creating $(HEROKUISH_RPM_PACKAGE_NAME)"
+	sudo fpm -t rpm -s dir -C /tmp/build -n herokuish \
+		-v $(HEROKUISH_VERSION) \
+		-a $(RPM_ARCHITECTURE) \
+		-p $(HEROKUISH_RPM_PACKAGE_NAME) \
+		--depends 'docker-engine >= 1.9.1' \
+		--depends 'sudo' \
+		--after-install rpm/herokuish.postinst \
+		--url "https://github.com/$(HEROKUISH_REPO_NAME)" \
+		--description $(HEROKUISH_DESCRIPTION) \
+		--license 'MIT License' \
+		.
+	mv *.rpm /tmp
+
+rpm-dokku:
+	rm -rf /tmp/tmp /tmp/build dokku_*_$(RPM_ARCHITECTURE).rpm
+	mkdir -p /tmp/tmp /tmp/build
+
+	mkdir -p /tmp/build/usr/bin
+	mkdir -p /tmp/build/usr/share/doc/dokku
+	mkdir -p /tmp/build/usr/share/dokku/contrib
+	mkdir -p /tmp/build/usr/share/lintian/overrides
+	mkdir -p /tmp/build/usr/share/man/man1
+	mkdir -p /tmp/build/var/lib/dokku/core-plugins/available
+
+	cp dokku /tmp/build/usr/bin
+	cp LICENSE /tmp/build/usr/share/doc/dokku/copyright
+	find . -name ".DS_Store" -depth -exec rm {} \;
+	cp -r plugins/* /tmp/build/var/lib/dokku/core-plugins/available
+	find plugins/ -mindepth 1 -maxdepth 1 -type d -printf '%f\n' | while read plugin; do touch /tmp/build/var/lib/dokku/core-plugins/available/$$plugin/.core; done
+	$(MAKE) help2man
+	$(MAKE) addman
+	cp /usr/local/share/man/man1/dokku.1 /tmp/build/usr/share/man/man1/dokku.1
+	gzip -9 /tmp/build/usr/share/man/man1/dokku.1
+	cp contrib/dokku-installer.py /tmp/build/usr/share/dokku/contrib
+	git describe --tags > /tmp/build/var/lib/dokku/VERSION
+	cat /tmp/build/var/lib/dokku/VERSION | cut -d '-' -f 1 | cut -d 'v' -f 2 > /tmp/build/var/lib/dokku/STABLE_VERSION
+ifneq (,$(findstring false,$(IS_RELEASE)))
+	sed -i.bak -e "s/^/`date +%s`:/" /tmp/build/var/lib/dokku/STABLE_VERSION && rm /tmp/build/var/lib/dokku/STABLE_VERSION.bak
+endif
+	git rev-parse HEAD > /tmp/build/var/lib/dokku/GIT_REV
+
+	@echo "-> Creating rpm package"
+	VERSION=$$(cat /tmp/build/var/lib/dokku/STABLE_VERSION); \
+	sudo fpm -t rpm -s dir -C /tmp/build -n dokku \
+		-v "$$VERSION" \
+		-a $(RPM_ARCHITECTURE) \
+		-p "dokku-$$VERSION-1.x86_64.rpm" \
+		--depends 'git' \
+		--depends 'make' \
+		--depends 'curl' \
+		--depends 'gcc' \
+		--depends 'man-db' \
+		--depends 'sshcommand' \
+		--depends 'gliderlabs-sigil' \
+		--depends 'docker-engine >= 1.9.1' \
+		--depends 'bind-utils' \
+		--depends 'nginx' \
```

Comment:
none. [EPEL](https://fedoraproject.org/wiki/EPEL) repository includes nginx 1.10.1. That is why I had to `yum install -y epel-release`